src/org/labkey/test/components/domain/DomainFormPanel.java

```diff
@@ -238,12 +248,12 @@ public String getPanelAlertText()
      */
     public WebElement getPanelAlertWebElement()
     {
+        getWrapper().waitFor(()-> BootstrapLocators.infoBanner.existsIn(getDriver()),
+                "the error alert did not appear as expected", 1000);
```

Comment:
Something to keep in mind about the waitFor call, when the wait expires it will throw an exception if there is a message parameter ("the error alert did not appear as expected.").
Adding this waitFor and message now means this function could throw when previously it did not.

Reply:
Good to know. Thanks. Looks like the only usage to date was in the SMAssayCreateTest, which I'll verify passes.